src/i_system.c

```diff
@@ -304,7 +304,13 @@ boolean I_GetMemoryValue(unsigned int offset, void *value, int size)
                         break;
                     }
 
-                    M_StrToInt(myargv[p], &val);
+                    if (!M_StrToInt(myargv[p], &val))
+                    {
+                        I_Error("Invalid parameter '%s' for -setmem, "
+                                "valid values are dos622, dos71, dosbox "
+                                "or memory offset.", myargv[p]);
```

Comment:
It's not an offset is it? I think it's up to 10 bytes of data.

Reply:
You're right, I'll correct it.